apps/nextjs-website/src/lib/fetch/navigation.ts

```diff
@@ -3,35 +3,160 @@ import { extractFromResponse } from './extractFromResponse';
 import { AppEnv } from '@/AppEnv';
 
 // Codec
+const HeroSectionCodec = t.strict({
+  __component: t.literal('sections.hero'),
+  title: t.string,
+  subtitle: t.union([t.string, t.null]),
+  theme: t.union([t.literal('light'), t.literal('dark')]),
+  inverse: t.boolean,
+  size: t.keyof({
+    small: null,
+    big: null,
+  }),
+  useHoverlay: t.boolean,
+  sectionID: t.union([t.string, t.null]),
+  image: t.union([
+    t.strict({
+      alternativeText: t.union([t.string, t.null]),
+      url: t.string,
+    }),
+    t.null,
+  ]),
+  background: t.union([
+    t.strict({
+      alternativeText: t.union([t.string, t.null]),
+      url: t.string,
+    }),
+    t.null,
+  ]),
+  ctaButtons: t.array(
+    // TODO: Replace with CTAButtonSchema when merged
+    t.intersection([
+      t.type({
+        text: t.string,
+        href: t.string,
+        variant: t.keyof({
+          text: null,
+          outlined: null,
+          contained: null,
+        }),
+        color: t.keyof({
+          inherit: null,
+          primary: null,
+          secondary: null,
+          success: null,
+          error: null,
+          info: null,
+          warning: null,
+        }),
+      }),
+      t.partial({
+        icon: t.union([t.string, t.null]),
+        size: t.keyof({
+          small: null,
+          medium: null,
+          large: null,
+        }),
+      }),
+    ])
+  ),
+});
+
+const EditorialSectionCodec = t.strict({
+  __component: t.literal('sections.editorial'),
+  title: t.string,
+  eyelet: t.union([t.string, t.null]),
+  body: t.string,
+  theme: t.union([t.literal('light'), t.literal('dark')]),
+  pattern: t.keyof({
+    // Move to separate Codec?
+    none: null,
+    dots: null,
+    solid: null,
+  }),
+  width: t.keyof({
+    // Move to separate Codec?
+    standard: null,
+    wide: null,
+    center: null,
+  }),
+  reversed: t.boolean,
+  sectionID: t.union([t.string, t.null]),
+  image: t.union([
+    t.strict({
+      alternativeText: t.union([t.string, t.null]),
+      url: t.string,
+    }),
+    t.null,
+  ]),
+  ctaButtons: t.array(
+    // TODO: Replace with CTAButtonSchema when merged
+    t.intersection([
+      t.type({
+        text: t.string,
+        href: t.string,
+        variant: t.keyof({
+          text: null,
+          outlined: null,
+          contained: null,
+        }),
+        color: t.keyof({
+          inherit: null,
+          primary: null,
+          secondary: null,
+          success: null,
+          error: null,
+          info: null,
+          warning: null,
+        }),
+      }),
+      t.partial({
+        icon: t.union([t.string, t.null]),
+        size: t.keyof({
+          small: null,
+          medium: null,
+          large: null,
+        }),
+      }),
+    ])
+  ),
+});
+
+const PageSectionCodec = t.union([HeroSectionCodec, EditorialSectionCodec]);
+
+const PageDataCodec = t.strict({
+  slug: t.string,
```

Comment:
Is the slug useful? I think it has the same value of `path` [^1]

[^1]: https://github.com/pagopa/b2b-portals/pull/95/files#diff-949f5b0650d3b0a5a059aef5735ce52b0e0ac2ab24f08ddcede7e12db951ffacR136

Reply:
It's been added with [this task](https://pagopa.atlassian.net/browse/B2BP-273?atlOrigin=eyJpIjoiNGQxY2U1MzRjMjkwNDgwZWI5ZDFmOGE3NjgxMDMwNDYiLCJwIjoiaiJ9) in mind.

We are going to have to substitute the use of `path` with `related.slug` soon and this reflects that.

Reply:
It is however currently unused, so I will remove it for the time being.